web/src/components/profileCreate/CreateFormTL.tsx

```diff
@@ -45,7 +45,7 @@ const CreateFormTL: React.FC = () => {
             {fields.map((name, index) => (
               <div key={name}>
                 <Flex flexDirection="row">
-                  <FormTitle style={{ margin: '14px 0 5px 0' }}>Technical Lead</FormTitle>
+                  <FormTitle style={{ margin: '14px 0 5px 0' }}>Technical Lead {index == 0 ? '(Primary)' : '(Secondary)'}</FormTitle>
```

Comment:
Normally we use `===` in javascript to make sure they fully match.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality

Reply:
You will need to update snapshot and make sure lints are fine by running:
```
in ../web run:
npm run test -- -u
Fir just as an FYI. the --u will update the snapshots captured during testing.
npm run lint -- --quiet -f codeframe --fix
npm run format -- --write
```